.lando.yml

```diff
@@ -11,6 +11,9 @@ services:
     composer:
       hirak/prestissimo: ^0.3.9
     overrides:
+      environment:
+        SIMPLETEST_DB: 'mysql://drupal8:drupal8@database/drupal8'
+        SIMPLETEST_BASE_URL: 'https://lando.lndo.site/'
```

Comment:
Should this be `http://localhost` similar to the ddev URL? There might be cases where `lando.lndo.site` might not be the valid URL of the site.

Reply:
To be safe, I'll change it